File: S01_AuthUI/app/src/main/java/com/example/s01_authui/MainActivity.java
package com.example.s01_authui;

import android.os.Bundle;
import android.view.View;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        //Cargar el layout activity_main.xml
        setContentView(R.layout.activity_main);
    }
}

File: S02_UNSFormRegister/app/src/main/java/com/example/s02_unsformregister/MainActivity.kt
package com.example.s02_unsformregister

import android.content.Intent
import android.graphics.RenderEffect
import android.graphics.Shader
import android.os.Build
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val imageView = findViewById<ImageView>(R.id.bgImage)

        imageView.post {
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
                val blur = RenderEffect.createBlurEffect(20f, 20f, Shader.TileMode.CLAMP)
                imageView.setRenderEffect(blur)
            }
        }


        val registerBtn = findViewById<Button>(R.id.register_btn)
        val viewRegistersBtn = findViewById<Button>(R.id.view_registers_btn)

        registerBtn.setOnClickListener {
            val intent = Intent(this@MainActivity, RegisterActivity::class.java)
            startActivity(intent)
            overridePendingTransition(R.anim.slide_in_up, R.anim.slide_out_up)
        }

        viewRegistersBtn.setOnClickListener {
            val intent = Intent(this@MainActivity, ViewStudentsActivity::class.java)
            startActivity(intent)
            overridePendingTransition(R.anim.slide_in_up, R.anim.slide_out_up)
        }
    }
}